twoliter/embedded/build.Dockerfile

```diff
@@ -138,11 +138,20 @@ RUN --mount=target=/host \
       KIT_REPOS+=("--repofrompath=${kit},/host/build/kits/${kit}/${ARCH}" --enablerepo "${kit}") ; \
     done && \
     echo "${KIT_REPOS[@]}" && \
+    declare -a EXTERNAL_KIT_REPOS && \
+    EXTERNAL_KITS=$(jq -c '.kit[]' < /host/build/external-kits/external-kit-metadata.json) && \
+    for kit in ${EXTERNAL_KITS} ; do \
+       KIT_NAME=$(jq '.vendor + "-" + .name' <<< "${kit}" | tr -d '"') && \
+       REPO_PATH=$(jq "\"/host/build/external-kits/\" + .vendor + \"/\" + .name + \"/${ARCH}\"" <<< "${kit}" | tr -d '"') && \
+       EXTERNAL_KIT_REPOS+=("--repofrompath=${KIT_NAME},${REPO_PATH}" --enablerepo "${KIT_NAME}"); \
+    done && \
+    echo "${EXTERNAL_KIT_REPOS[@]}" && \
```

Comment:
Is this just for debugging?

Reply:
Just following the pattern for previous block

Reply:
I think it was for debugging in the previous block. Possibly removing it was forgotten. It seems like it could come in handy. I could go either way on removing it.